### ISSUE TITLE
How do I show a picture like you that is all in one category？

### ISSUE THREAD

**Issue by LittleStars666** [Hydrino/ACGAN_cifar10#2]
The pictures I trained showed ten different categories of a picture，but I want to save them one by one. Is there any way ？

**Comment by Hydrino** [comment]
`labels = 4*torch.ones((10,),dtype = torch.long, device = device,)
fakes = gen(noise,labels).detach()
showImage(make_grid(fakes))`

replace the '4' with the category you want to show pictures of.

**Comment by LittleStars666** [comment]
I tried, and the result was the same
